src/utils/auditorium.ts

```diff
@@ -0,0 +1,16 @@
+export interface GetPositionHashProps {
+  row: number;
+  column: number;
+}
+
+export const getPositionHash = ({ row, column }: GetPositionHashProps) => {
+  return `${row}|${column}`;
+};
+
+export interface TranslateIndexProps {
+  index: number;
+  totalAmount: number;
+}
+
+export const translateIndex = ({ index, totalAmount }: TranslateIndexProps) =>
+  index - Math.floor(totalAmount / 2);
```

Comment:
It might be good to add some comments to this, as it's not immediately obvious what the benefit/etc of it is.

Possibly there is a better/more descriptive name we could use for the function too? I believe it basically converts one type of coordinates (eg. screen coordinates) to another (eg. cartesian coordinates) or similar.

Reply:
Any suggestions? Name + Comment sound good

Reply:
Not off the top of my head, knowing exactly what it is used for/doing would be where I would start, then pick a name that aligns to/heps to imply that.

This was my best guess, but you've been far more involved in this code far more recently:

> I believe it basically converts one type of coordinates (eg. screen coordinates) to another (eg. cartesian coordinates) or similar.

I would suggest looking at where/how it's used + what it does, then using that to derive the name. Sometimes explaining what it does in a few sentences/paragraph or so can make that more obvious/'come to light'.

Reply:
Renamed, but still don't know how to describe the function https://github.com/sparkletown/sparkle/pull/1251/commits/dfa29a8255b81916d6f9213b02a52198ba1c1f2b